playwright-e2e/tests/judicial-message.spec.ts

```diff
@@ -8,7 +8,7 @@ import { newSwanseaLocalAuthorityUserOne,CTSCUser ,judgeUser} from '../settings/
 import { expect } from '@playwright/test';
 
 
-test.describe('send and reply message',()=>{
+test.describe('send and reply message @nightly',()=>{
```

Comment:
do we need nightly tag?

Reply:
They will not run otherwise. As I created a new tag for accessibility and e2e tests that are for nightly. Please see the scripts in package.json

Reply:
I thought this test is running even without the tag? @nightly tag was used for codecept tests if I'm not mistaken. 
https://build.hmcts.net/job/HMCTS_d_to_i_Nightly/job/fpl-ccd-configuration/job/master/781/UI_20Functional_20Test_20Report/

Reply:
Removing the Codeept tests was done after I branched. I have removed 'nightly' and kept the removal of the Codecept tests. But kept accessibility tags, as we have to configure the test(s) to use the axe fixture. We may want to pick and chose which tests we accessibility test?